components/lists.html

```diff
@@ -100,32 +100,130 @@
         <!-- Main Content Section Start -->
         <section class="doc-content">
             <h2 class="title">List</h2>
-            <div class="subtitle">Typography is the art and technique of arranging letters and characters known as type. It also refers to different elements of typography including the size of the letters, line length and the spacing on a line or the entire page.</div>
+            <div class="subtitle">Lists can be used to display information in points not in a single paragraph, like notifications etc.</div>
 
-            <h3 class="title">Headings</h3>
-            <div class="subtitle">All HTML headings, from h1 through h6, are available.</div>
+            <h3 class="title">Simple List</h3>
+            <div class="subtitle">Simple list are of two types either ordered list or unordered list.</div>
```

Comment:
Nice work on UI. Just one suggestion, I think, you can use `<p>`  for description.

Reply:
Ok Himanshu, thanks for suggesting noted your suggestion. I have to update all the components for this will do this in the last component.